app/routes/users/components/UserProfile/AchievementsBox.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, Flex } from '@webkom/lego-bricks';
+import { Trophy } from 'lucide-react';
+import moment from 'moment-timezone';
+import { useState } from 'react';
+import Tooltip from 'app/components/Tooltip';
+import AchievementsInfo, {
+  rarityToColorMap,
+} from 'app/utils/achievementConstants';
+import styles from './UserProfile.css';
+import type { Achievement } from 'app/store/models/User';
+
+export const AchievementsBox = ({
+  achievements,
+}: {
+  achievements: Achievement[];
+}) => {
+  const [showAll, setShowAll] = useState(false);
+  const topAchievements = [...achievements]
+    .sort(
+      (a, b) =>
+        AchievementsInfo[b.identifier][b.level].rarity -
+        AchievementsInfo[a.identifier][a.level].rarity,
+    )
+    .slice(0, showAll ? 999 : 5);
+  return (
+    <Flex
+      column
+      width={'100%'}
+      alignItems="center"
+      justifyContent="space-between"
+    >
+      <Flex width={'100%'} alignItems="center" justifyContent="center">
+        <h2>Trofeer</h2>
+      </Flex>
```

Comment:
Not a fan of the centered header (at least on larger viewports), since all other headers on the page are left-sided